apps/web/app/components/FileUploadInput/ImageUploadInput.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+
+import { Icon } from "../Icon";
+
+interface ImageUploadProps {
+  field: { value?: string };
+  handleImageUpload: (file: File) => void;
+  isUploading: boolean;
+  imageUrl?: string;
+}
+
+const ImageUploadInput = ({ field, handleImageUpload, isUploading }: ImageUploadProps) => {
+  return (
+    <div className="flex items-center justify-center flex-col gap-y-2">
+      <div className="w-full h-[20rem] rounded-lg border-solid border-2 border-gray-300 flex justify-center items-center bg-gray-100 relative cursor-pointer overflow-hidden flex-col">
+        {field.value && (
+          <img src={field.value} alt="Uploaded" className="w-full h-full object-cover" />
+        )}
+        <div
+          className={`absolute inset-0 flex flex-col items-center justify-center text-center ${
+            field.value ? "text-white" : ""
+          }`}
+        >
+          <Icon name="UploadImageIcon" />
+
+          <div className="flex items-center justify-center mt-2">
+            <span className={`text-lg font-semibold text-[#7CA3DE]`}>
+              {field.value ? "Click to replace" : "Click to upload"}
+            </span>
+            <span className="ml-2 text-lg font-semibold">or drag and drop</span>
+          </div>
+
+          <div
+            className={`text-sm w-full px-2 mt-2 ${field.value ? "text-white" : "text-gray-600"}`}
```

Comment:
Same case as above, please use `cn` function 😉

Reply:
I've already added it on another branch so it will be added with the next merge